bin/find_scheme.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+
+xcodebuild -list -project "$@" \
+  | awk '/Schemes:/ { getline; print }' \
```

Comment:
What does `getline` do? Is this line just printing lines that have `/Schemes:/` in them?